ts/interactions/conversations/unsendingInteractions.ts

```diff
@@ -398,6 +401,9 @@ export async function deleteMessagesById(messageIds: Array<string>, conversation
         window.inboxStore?.dispatch(resetRightOverlayMode());
       },
       closeAfterInput: false,
+      onClickClose: () => {
+        window.inboxStore?.dispatch(updateConfirmModal(null));
```

Comment:
closeDialog() ?

Reply:
closeDialog was not declared in that function so I've created it